lib/common/xpath.c

```diff
@@ -266,37 +266,68 @@ get_xpath_object(const char *xpath, xmlNode * xml_obj, int error_level)
     return result;
 }
 
-int
-pcmk__element_xpath(const xmlNode *xml, char *buffer, int offset,
-                    size_t buffer_size)
+/*!
+ * \internal
+ * \brief Get an XPath string that matches an XML element as closely as possible
+ *
+ * \param[in] xml  The XML element for which to build an XPath string
+ *
+ * \return A \p GString that matches \p xml, or \p NULL if \p xml is \p NULL.
+ *
+ * \note The caller is responsible for freeing the string.
```

Comment:
It's worthwhile in such notes to give the function that must be used, e.g. "using g_string_free()"

Reply:
Done in next update